Realm/RLMObjectStore.mm

```diff
@@ -517,15 +519,21 @@ id RLMGetObject(RLMRealm *realm, NSString *objectClassName, id key) {
         return nil;
     }
 
-    return RLMCreateObjectAccessor(realm, objectSchema, row);
+    return RLMCreateObjectAccessor(realm, objectSchema, (*objectSchema.table)[row]);
```

Comment:
I think I started removing that overload, then discovered that it was actually noticeably nicer in most of the places where it's used since the `objectSchema` is often an expensive expression that'd otherwise need to be cached. I'll switch this one back to just passing this index.